src/components/modal/content.js

```diff
@@ -0,0 +1,29 @@
+import React, { forwardRef } from 'react'
+import PropTypes from 'prop-types'
+
```

Comment:
_minor_: 2 new lines after import, & the same for `modal/footer.js` `modal/header.js`